src/shared/utils/helpers/index.ts

```diff
@@ -18,7 +18,6 @@ import numToSI from "./num-to-si";
 import poll from "./poll";
 import randomStr from "./random-str";
 import removeAuthParam from "./remove-auth-param";
-import returnStringFromString from "./return-str";
```

Comment:
What is this util? Is it even used? If not, the file it's from should be deleted.